mythril/mythril/mythril_disassembler.py

```diff
@@ -0,0 +1,298 @@
+import logging
+import re
+import solc
+import os
+
+from ethereum import utils
+from solc.exceptions import SolcError
+from typing import List, Tuple, Optional
+from mythril.ethereum import util
+from mythril.ethereum.interface.rpc.client import EthJsonRpc
+from mythril.exceptions import CriticalError, CompilerError, NoContractFoundError
+from mythril.support import signatures
+from mythril.support.truffle import analyze_truffle_project
+from mythril.ethereum.evmcontract import EVMContract
+from mythril.ethereum.interface.rpc.exceptions import ConnectionError
+from mythril.solidity.soliditycontract import SolidityContract, get_contracts_from_file
+
+log = logging.getLogger(__name__)
+
+
+class MythrilDisassembler:
```

Comment:
I think we should come up with a more precise name later. 
Also docstring 😇

Reply:
bump

Reply:
done :+1: